_artifacts/logstash-oss-with-opensearch-output-plugin/logstash-oss-with-opensearch-output-plugin-7.16.3-docker-arm64.markdown

```diff
@@ -7,5 +7,15 @@ architecture: arm64
 slug: logstash-oss-with-opensearch-output-plugin-7.16.3-docker-arm64
 category: logstash-oss-with-opensearch-output-plugin
 type: docker_hub
-link: https://hub.docker.com/r/opensearchproject/logstash-oss-with-opensearch-output-plugin
+inline_instructions:
+- label: "Docker Hub"
+  code: docker pull opensearchproject/logstash-oss-with-opensearch-output-plugin:7.16.3
```

Comment:
It might be better to have this just be a URL. Then we can have one place generate the code using the `type` I suggested above and the `url`.

e.g.

```
url: opensearchproject/logstash-oss-with-opensearch-output-plugin:7.16.3
```

Reply:
The intention is to make this simple enough to be scalable in any direction the maintainers choose. By defining types and making it strict, that flexibility is removed. Also, these files are not created by hand and are generated by a script.

Reply:
For Data Prepper, we are still making these by hand. But, we should probably find a way to automate this as well.